forge-gui/res/cardsfolder/upcoming/birthing_ritual.txt

```diff
@@ -0,0 +1,9 @@
+Name:Birthing Ritual
+ManaCost:1 G
+Types:Enchantment
+T:Mode$ Phase | TriggerZones$ Battlefield | Phase$ End of Turn | ValidPlayer$ You | IsPresent$ Creature.YouCtrl | Execute$ TrigPeek | TriggerDescription$ At the beginning of your end step, if you control a creature, look at the top seven cards of your library. Then you may sacrifice a creature. If you do, you may put a creature card with mana value X or less from among those cards onto the battlefield, where X is 1 plus the sacrificed creature’s mana value. Put the rest on the bottom of your library in a random order.
+SVar:TrigPeek:DB$ PeekAndReveal | Defined$ You | NoReveal$ True | RememberPeeked$ True | PeekAmount$ 7 | SubAbility$ DBChangeZone
+SVar:DBChangeZone:DB$ ChangeZone | UnlessCost$ Sac<1/Creature> | UnlessPayer$ You | UnlessSwitched$ True | Origin$ Library | Destination$ Battlefield | ChangeType$ Creature.cmcLEX+IsRemembered | ChangeNum$ 1
```

Comment:
It doesn't let me choose among the looked at cards, it just shows my library

Reply:
you might have to write it without cost for now

Reply:
Thanks got it working now

Reply:
somehow I have doubts

Reply:
What do you mean ?

Reply:
I mean it doesn't:
![image](https://github.com/Card-Forge/forge/assets/8506892/e8af318c-9ee4-4c8a-bfd2-6ff6375c030e)

Reply:
Ah I see, forgot the condition that you did sack a creature ?

Reply:
now the problem is you still need to put the rest on the bottom...

Reply:
If I read this card correctly, the card should stay on top if you not sacrifice a creatue ?

Reply:
I was just about to explain this in detail but this was already answered elsewhere anyway:
https://www.reddit.com/r/mtgrules/comments/1d0i8po/birthing_ritual_wording/

Reply:
1. instead of extra code you can just switch the PeekAndReveal to use Imprinted
2. you're still not making sure the Dig doesn't move the rest

Reply:
I think there's only ImprintRevealed ? non imprintpeeked ?

Reply:
Ugh yea sorry, what a messy card

I suggest you only Remember the sacrifice
then you can replace the ChangeZoneAll for another Dig effect with inverted Condition for Remembered + `ChangeNum$ 0` so they all get moved to bottom